Documentation/gettingstarted/bird.rst

```diff
@@ -0,0 +1,306 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+
+****************************
+Using BIRD to run BGP
+****************************
+
+`BIRD is an open-source implementation for routing Internet Protocol
+packets on Unix-like operating systems <https://en.wikipedia.org/wiki/Bird_Internet_routing_daemon>`_.
+If you are not familiar with it, you had best have a glance at the `User's Guide
+<https://bird.network.cz/?get_doc&f=bird.html&v=20>`_ first.
+
+`BIRD <https://bird.network.cz>`_ maintains two release families at present:
+``1.x`` and ``2.x``, and the configuration format varies a lot between them.
+Unless you have already deployed the ``1.x``, we suggest using ``2.x``
+directly, as the ``2.x`` will live longer. In the following, we will denote
+``bird`` as the ``bird2`` software.
+
+This guide shows how to install and configure bird on CentOS 7.x to make it
+collaborate with Cilium. Installation and configuration on other platforms
+should be very similar.
+
+Install bird
+##################
+
+.. code:: bash
+
+    $ yum install -y bird2
+
+    $ systemctl enable bird
+    $ systemctl restart bird
+
+
+Test the installation:
+
+.. code:: bash
+
+    $ birdc show route
+    BIRD 2.0.6 ready.
+
+    $ birdc              # interactive shell
+    BIRD 2.0.6 ready.
+    bird> show bfd sessions
+    There is no BFD protocol running
+    bird>
+    bird> show protocols all
+    Name       Proto      Table      State  Since         Info
+    device1    Device     ---        up     10:53:40.147
+
+    direct1    Direct     ---        down   10:53:40.147
+      Channel ipv4
+        State:          DOWN
+        Input filter:   ACCEPT
+        Output filter:  REJECT
+    ...
+
+Basic configuration
+#####################
+
+It's hard to discuss bird configurations without considering specific BGP
+schemes. However, BGP scheme design is beyond the scope of this guide.
+If you are interested in this topic, refer to `BGP in the Data Center
+<https://www.oreilly.com/library/view/bgp-in-the/9781491983416/>`_ (O'Reilly,
+2017) for a quick start.
+
+In the following, we will restrict our BGP scenario as follows:
+
+.. image:: images/bird_sample_topo.png
+   :scale: 35%
```

Comment:
This is what it looks like with 35%:
![bird-guide-small-images](https://user-images.githubusercontent.com/1764210/75355553-f703da00-58ae-11ea-9c67-b2f8c150ef3d.png)
<br>
And this is with 70%:
![bird-guide-larger-images](https://user-images.githubusercontent.com/1764210/75355579-05ea8c80-58af-11ea-95bf-7eb827614df8.png)

Reply:
`70%` is much better, i'll preview the rendering results with `make render-docs` next time :)

Reply:
Sorry, i thought the sizes are already `70%` in `bird.rst`, i just corrected them and pushed again.